MountainArr_java_Pratheeksha

```diff
@@ -0,0 +1,35 @@
+import java.io.*;
+import java.lang.*;
+import java.util.*;
+
+class GFG {
+
+// Function to check if the given array
+// is mountain array or not
+static Boolean isMountainArray(int arr[])
+{
+	if (arr.length < 3)
+		return false;
+	int flag = 0, i = 0;
+	for (i = 1; i < arr.length; i++)
+		if (arr[i] <= arr[i - 1])
+			break;
+
+	if (i == arr.length || i == 1)
+		return false;
+
+	for (; i < arr.length; i++)
```

Comment:
if we don't have any initial block for variable declaration, can we use a `while` loop here?